packages/modals/src/elements/Modal.js

```diff
@@ -132,6 +132,10 @@ Modal.propTypes = {
    * @param {Object} event - DOM event that triggered the close action
    */
   onClose: PropTypes.func,
+  /**
+   * The DOM element where the modal will be rendered to
+   **/
+  container: PropTypes.instanceOf(Element),
```

Comment:
Could we rename this prop to `appendToNode`? We use this style in other components where it is optional, might make sense to continue it here.